test/spmd/tensor/test_pointwise_ops.py

```diff
@@ -10,50 +13,182 @@
 
 from spmd.tensor.ops.pointwise_ops import pointwise_rule
 from spmd import DeviceMesh, DTensor
-from spmd.tensor.placement_types import Shard, Replicate, _Partial, DTensorSpec
+from spmd.tensor.placement_types import (
+    Shard,
+    Replicate,
+    _Partial,
+    DTensorSpec,
+    Placement,
+)
 from torch.distributed.distributed_c10d import ReduceOp
 from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
 
 
+def deepcopy_convert_to_dtensor(
+    val: Any,
+    device_mesh: DeviceMesh,
+    placements: Sequence[Placement],
+) -> Any:
+    """
+    Recursively coerce (over Sequence and Dict types) Tensors into DTensors.
+
+    :param device_mesh: the DeviceMesh to use.
+    :param placements: the Placement list to use.
+    :return: the transformed structure.
+    """
+    if isinstance(val, Tensor):
+        return DTensor(
```

Comment:
could you use `distribute_tensor` API instead? https://github.com/pytorch/tau/blob/main/spmd/tensor/__init__.py#L14 This is a more legit API to use (i.e. shard an existing torch.Tensor to different ranks), instead of creating a DTensor from each rank's local tensor.

Reply:
done.

Reply:
@wanchaol Should we add a comment about what is the suggested way to create a DTensor and what the benefits of using distribute_tensor are? I feel I've run into this issue of using DTensor directly as well.

Reply:
yeah sure, let me update the comment of Tensor constructor to reflect this.